Fix : NoMongo: Figma - Admin Profile Page Violates The Figma Style Guide #3172

AryanSwaroop opened the pull request:
**What kind of change does this PR introduce?**

BugFix

**Issue Number:**

Fixes #3172

**Snapshots/Videos:**

https://github.com/user-attachments/assets/f551b8b2-2293-4ce8-b951-8f75fcf87ca2

**If relevant, did you update the documentation?**

No

**Summary**

Changed the design objects

**Does this PR introduce a breaking change?**

No

## Checklist

### CodeRabbit AI Review
- [ ] I have reviewed and addressed all critical issues flagged by CodeRabbit AI
- [ ] I have implemented or provided justification for each non-critical suggestion
- [ ] I have documented my reasoning in the PR comments where CodeRabbit AI suggestions were not implemented

### Test Coverage
- [ ] I have written tests for all new changes/features
- [ ] I have verified that test coverage meets or exceeds 95%
- [ ] I have run the test suite locally and all tests pass


**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced an automated pull request review process that requires CodeRabbit’s approval before merging.

- **Chores**
  - Expanded the review configuration to treat index.html as a sensitive file.
  - Streamlined the workflow by removing legacy approval validation steps and related scripts.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Cioppolo14 commented:
@coderabbitai full review

Cioppolo14 commented:
@AryanSwaroop I reran coderabbit, please make review all the failed checks and address them, and any comments coderabbit has.

AryanSwaroop commented:
working on changes

AryanSwaroop commented:
![image](https://github.com/user-attachments/assets/21f4ccdd-68c8-468a-8739-82c50d489a18)

This are the changes , in this push , i'll be adding more , guide for any more changes

palisadoes commented:
> ![image](https://private-user-images.githubusercontent.com/113257605/423152962-21f4ccdd-68c8-468a-8739-82c50d489a18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNjQ3MjIsIm5iZiI6MTc0MjA2NDQyMiwicGF0aCI6Ii8xMTMyNTc2MDUvNDIzMTUyOTYyLTIxZjRjY2RkLTY4YzgtNDY4YS04NzM5LTgyYzUwZDQ4OWExOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMxNVQxODQ3MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00Mjg1Njc2N2FhMDIzOTVjNWIwOGE4ZTdkNDVlMjllNWRhMWYxOTE4MGNiODQxZmQ2NjllODdkNDZkZGU3ZDViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._a4p0hlCW86FVvrfWXmyaszWoXTBdc6HPVPc2cCaUC8)
> 
> This are the changes , in this push , i'll be adding more , guide for any more changes

Your changes do not match that of Figma.

palisadoes commented:
> Your work doesn't match the expected output of the issue. Please fix.
> 
>     * [NoMongo: Figma: Admin Profile Page Violates The Figma Style Guide #3172](https://github.com/PalisadoesFoundation/talawa-admin/issues/3172)
> 
> 
> ![image](https://private-user-images.githubusercontent.com/16875803/422429519-6350752b-5994-4e59-ab89-f5bf9522a3a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNjQ3MjMsIm5iZiI6MTc0MjA2NDQyMywicGF0aCI6Ii8xNjg3NTgwMy80MjI0Mjk1MTktNjM1MDc1MmItNTk5NC00ZTU5LWFiODktZjViZjk1MjJhM2EzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE4NDcwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5MjUyN2IyNjQxODhjMzc0NzdiNjhkMzEzZGM2MzRlNTZjMGJiZjFkYjBmMmIxMzhiNDY1YWVlMTI1MWUzMDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.nBD79lWUIUirwccLLp7nn1KsCooRp_qCtlqzoqiLCdw)

1. This is what the output should look like
2. It is detailed extensively in the issue

AryanSwaroop commented:
![image](https://github.com/user-attachments/assets/9a04b56d-e603-49b2-9fb1-091c7a12aefa)

![image](https://github.com/user-attachments/assets/17723207-b881-45c3-82f9-1430cf094f90)

Please review should this be considered final design ? followed figma design and your suggestion regarding profile photo @palisadoes

AryanSwaroop commented:
@coderabbitai full review

AryanSwaroop commented:
@coderabbitai full review

palisadoes commented on src/style/app-fixed.module.css:
```diff
@@ -6295,6 +6298,32 @@ form {
+  border: solid;
+  color: var(--featuresBox-border);
+  border-width: 1px;
+  background-color: #ffffff;
```
Use a variable.

palisadoes reviewed (changes requested):
See comment

AryanSwaroop commented:
> 1. Please make coderabbit.ai approves your work.
> 
>     2. Resolve any conflicting files
> 
>     3. Make sure all tests pass and are valid.
> 
>     4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

sure

palisadoes commented:
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

AryanSwaroop commented:
@coderabbitai full review

AryanSwaroop commented:
@palisadoes requested changes from coderabbit are suggesting to add boolean to a string object that's redundant , i have updated the objects for actions. please review

AryanSwaroop commented:
@CodeRabbitAI Please re-review this PR. I’ve made the necessary changes based on the previous feedback.

palisadoes commented:
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

@coderabbitai full review

palisadoes commented:
@coderabbitai full review

palisadoes commented:
@coderabbitai full review

palisadoes commented:
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

AryanSwaroop commented:
@coderabbitai re review

AryanSwaroop commented:
@coderabbitai full review

palisadoes reviewed (changes requested):
1. You have dozens of recommended CodeRabbit.ai changes that you have marked as resolved without making any effort to make the suggested changes.
2. Please address ALL the CodeRabbit.ai suggestions and don't dismiss them in such a cavalier manner.
3. We need to ensure that the quality of our code base is maintained

palisadoes reviewed (changes requested):
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

palisadoes commented:
I'm going to close this PR. Not much progress has been made in 3 weeks.

1. Almost 300 CodeRabbit comments
2. 61 commits

This is an important feature to implement. I'm going to assign it to someone else.

![image](https://github.com/user-attachments/assets/bd3e54bc-c2cd-4146-a145-97427fe098ed)

AryanSwaroop commented:
@palisadoes there were too many features to implement , I would have completed it by today.

palisadoes commented:
Will you be able to submit the updates this weekend?

palisadoes commented:
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

palisadoes commented:
@coderabbitai full review

palisadoes reviewed (changes requested):
Remove file `er` from the root directory

palisadoes commented:
## Add a video of the latest UI/UX

Your previous videos reflect an old design that does not conform to the expectations of the issue.

1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.

palisadoes reviewed (changes requested):
## Add a video of the latest UI/UX

Your previous videos reflect an old design that does not conform to the expectations of the issue.

- #3172 

![image](https://github.com/user-attachments/assets/6350752b-5994-4e59-ab89-f5bf9522a3a3)

AryanSwaroop commented:
https://github.com/user-attachments/assets/6754f079-a154-498f-ac33-c4fbda4481c6

This is the working UI at the moment

palisadoes commented:
![image](https://github.com/user-attachments/assets/22093d43-bb67-42ba-9a25-0ea5d8f3599d)

palisadoes reviewed (changes requested):
![image](https://github.com/user-attachments/assets/22093d43-bb67-42ba-9a25-0ea5d8f3599d)

![image](https://github.com/user-attachments/assets/6344d306-1ade-4be0-8ee9-85e1dab7fa6e)

![image](https://github.com/user-attachments/assets/eee89571-116a-4bec-85f5-8f5fe0fdad48)

AryanSwaroop commented:
![image](https://github.com/user-attachments/assets/f11a6d9b-99fd-4d06-aa3d-971746e2ff23)

fixed spacing on reset buttons 

![image](https://github.com/user-attachments/assets/0c8b8e0f-80c5-42ec-8d25-daa225734438)

fixed fixes for details section

AryanSwaroop commented:
@coderabbitai full review

palisadoes commented:
@noman2002 @duplixx @Dante291 PTAL

Most of the files are auto-documentation and icons.

palisadoes commented:
1. Please make coderabbit.ai approves your work.
2. Resolve any conflicting files
3. Make sure all tests pass and are valid.
4. Ensure the test code coverage for your patch reaches as close to **100%** as possible.
